src/textual/widget.py

```diff
@@ -32,15 +32,18 @@
     ConsoleRenderable,
     JustifyMethod,
     RenderableType,
-    RenderResult,
-    RichCast,
 )
+from rich.console import RenderResult as RichRenderResult
```

Comment:
Why are these two imports separated from the other `rich.console` imports?
Why not the suggestion I show below?

```py
from rich.console import (
    Console,
    ConsoleOptions,
    ConsoleRenderable,
    JustifyMethod,
    RenderableType,
    RenderResult as RichRenderResult,
    RichCast,
)
```

Reply:
I'll let you have that argument with black and/or whatever else it is that rewrote my code from that.

Reply:
Ah ok, I understand 🤣